dbt/adapters/athena/impl.py

```diff
@@ -186,10 +187,9 @@ def generate_s3_location(
         return mapping[s3_data_naming]
 
     @available
-    def get_glue_table_location(self, relation: AthenaRelation) -> Optional[str]:
+    def get_glue_table(self, relation: AthenaRelation) -> Optional[str]:
```

Comment:
```suggestion
    def get_glue_table(self, relation: AthenaRelation) -> Optional[GetTableResponseTypeDef]:
```
see `mypy_boto3_glue.type_defs`